src/main/app/src/app/shared/edit/edit-datum-groep/edit-datum-groep.component.ts

```diff
@@ -100,16 +103,22 @@ export class EditDatumGroepComponent extends EditComponent implements OnInit {
 
     validate(): void {
         const start: Moment = moment(this.startDatumField.formControl.value);
-        const streef: Moment = moment(this.streefDatumField.formControl.value);
         const fatale: Moment = moment(this.fataleDatumField.formControl.value);
-        this.showStreefError = streef.isBefore(start);
-        this.showFataleError = fatale.isBefore(streef);
+        if (this.streefDatumField.formControl.value) {
```

Comment:
In zaak-view.component.ts worden de termen uiterlijkeEinddatumAfdoening en einddatumGepland gebruikt i.p.v. streefDatum en fataleDatum.